Installing from github requires yarn to be available

@giltho, I am not sure if this is an issue. When installing this package from github `npm install giltho/gatsby-transformer-bibtex#master`, I go the following error:

```
npm ERR! prepareGitDep 1> 
npm ERR! prepareGitDep > gatsby-transformer-bibtex@0.0.3 prepare /home/grosser/.npm/_cacache/tmp/git-clone-ebd4b032
npm ERR! prepareGitDep > cross-env NODE_ENV=production yarn build
npm ERR! prepareGitDep 
npm ERR! prepareGitDep 
npm ERR! prepareGitDep 2> npm WARN install Usage of the `--dev` option is deprecated. Use `--only=dev` instead.
npm ERR! prepareGitDep npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm ERR! prepareGitDep npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm ERR! prepareGitDep npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm ERR! prepareGitDep npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! prepareGitDep npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm ERR! prepareGitDep npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm ERR! prepareGitDep npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
npm ERR! prepareGitDep ERROR: [Errno 2] No such file or directory: 'build'
npm ERR! prepareGitDep npm ERR! code ELIFECYCLE
npm ERR! prepareGitDep npm ERR! errno 1
npm ERR! prepareGitDep npm ERR! gatsby-transformer-bibtex@0.0.3 prepare: `cross-env NODE_ENV=production yarn build`
npm ERR! prepareGitDep npm ERR! Exit status 1
npm ERR! prepareGitDep npm ERR! 
npm ERR! prepareGitDep npm ERR! Failed at the gatsby-transformer-bibtex@0.0.3 prepare script.
npm ERR! prepareGitDep npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! prepareGitDep 
npm ERR! prepareGitDep npm ERR! A complete log of this run can be found in:
npm ERR! prepareGitDep npm ERR!     /home/grosser/.npm/_logs/2020-05-11T05_10_30_160Z-debug.log
```

It did not provide any error message that was helpful to me. After searching for a while, I found that the problem was me missing 'yarn' on my system. Yarn is used in the prepare statement of this package. I am not sure if there are ways to drop this dependency, but as yarn is a non-standard package on ubuntu, I wanted to make sure this issue is recorded somewhere. Feel free to close this issue if there is nothing to be fixed in your repo.

--- a/package.json
+++ b/package.json
@@ -26,8 +26,8 @@
   },
   "scripts": {
     "build": "babel src --out-dir .",
-    "prepublish": "yarn build",
-    "prepare": "cross-env NODE_ENV=production yarn build",
+    "prepublish": "npm build",
+    "prepare": "cross-env NODE_ENV=production npm build",
     "watch": "babel -w src --out-dir . "
   }
 }

